toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/i18n/UIMessages.java

```diff
@@ -0,0 +1,26 @@
+package io.toast.tk.agent.ui.i18n;
+
+
+public class UIMessages {
+
+    private UIMessages(){}
+
+    public static final String PROXY_ADDR = "Proxy address";
+    public static final String PROXY_PORT = "Proxy port";
+    public static final String PROXY_USER = "Proxy User Name";
+    public static final String PROXY_PWD = "Proxy User Password";
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2068)